site/lazy-queues.md

```diff
@@ -0,0 +1,244 @@
+# Lazy Queues
+
+## <a id="overview" class="anchor" /> [Overview](#overview)
+
+Since **RabbitMQ 3.6.0**, the broker has the concept of **Lazy Queues** -
+queues that try to keep as many messages as possible on disk,
+and only load them in RAM when requested by consumers,
+therefore the lazy denomination.
+
+One of the main goals of lazy queues is to be able to support very long queues (many millions of messages).
+Queues can become very long for various reasons:
+
+<ul class="plain">
+  <li>consumers are offline / have crashed / are down for maintenance</li>
+  <li>there is a sudden message ingress spike, producers are outpacing consumers</li>
+  <li>consumers are simply slower than normal</li>
+</ul>
+
+By default, queues keep an in-memory cache of messages that's filled up as messages are published into RabbitMQ.
+The idea of this cache is to be able to deliver messages to consumers as fast as possible -
+note that persistent messages are written to disk as they enter the broker **and** kept in this cache at the same time.
```

Comment:
I suggest ending the previous sentence "as fast as possible." then start the new one "Note that..."